packages/examples/packages/jsx/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "baseUrl": "./",
+    "jsx": "react-jsxdev",
```

Comment:
Shouldn't this be pointing at our JSX runtime?

Reply:
It is through the parent `tsconfig`.